package.json

```diff
@@ -11,12 +11,7 @@
     "emoticon",
     "jquery-embed"
   ],
-  "main": [
-    "./src/jquery.embed.js",
-    "./src/jquery.embed.css",
-    "./fonts/*",
-    "./images/*"
-  ],
+  "main": "./src/jquery.embed.js",
```

Comment:
Why do we need to remove the css file from main. It will create problems as the css file won't be added automatically while using **grunt-wiredep**